release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -210,6 +210,10 @@ For more information, see xref:../installing/installing_with_agent_based_install
 You can perform an Agent-based installation in a disconnected environment. To create an image that is used in a disconnected environment, the `imageContentSources` section in the `install-config.yaml` file must contain the mirror information or `registries.conf` file if you are using ZTP manifests. The actual configuration settings to use in these files are supplied by either the `oc adm release mirror` or `oc mirror` command.
 For more information, see xref:../installing/installing_with_agent_based_installer/understanding-disconnected-installation-mirroring.adoc#understanding-disconnected-installation-mirroring[Understanding disconnected installation mirroring].
 
+[id="ocp-4-12-graph-true-field-disconnected"]
+==== Explanation of a `graph: true` field when creating the image set configuration
```

Comment:
Per the contributing guides, Do not use backticks or other markup in assembly or module headings. I verified with the peer review squad D and this also applies to release note headings. Maybe instead you could use "Explanation of field to build and push graph-data"?

Reply:
Agree and addressed. Thanks :)